mpc_controller/SinglePolyAC[] trajs